install-gui.sh

```diff
@@ -33,7 +33,14 @@ if [ "$(uname)" = "Linux" ]; then
 					echo >&2 "Please install NODEJS&NPM manually"
 			}
 		else
-			sudo apt-get install -y npm nodejs libxss1
+			program_list=( "libxss1" )
+			if ! [ -x "$(command -v npm)" ]; then
+			    prorgam_list+=( "npm" )
+			fi
+			if ! [ -x "$(command -v node)" ]; then
+			    program_list+=( "nodejs" )
+			fi
+			sudo apt-get install -y "${program_list[@]}"
```

Comment:
Even if `nodejs` is already installed, `sudo apt-get` tries to run whatever.
Ideally if nodejs/npm are installed already, `apt-get` don't need to be run at all.

By the way, we're planning a bunch of updates to the `install-gui.sh`.
I tried to rebase the upcoming PR onto your branch to express our-thanks but turned out number of conflicts are required to be resolved.

So I've decided to manage the updates to `install-gui.sh` in https://github.com/Chia-Network/chia-blockchain/pull/9921
Much appreciate your work and your patience.